src/printer.js

```diff
@@ -332,8 +332,9 @@ function genericPrintNoParens(path, options, print, args) {
       ]);
     case "FunctionDeclaration":
     case "FunctionExpression":
+    case "TSNamespaceFunctionDeclaration":
       if (isNodeStartingWithDeclare(n, options)) {
-        parts.push("declare ");
+        parts.push(hardline, "declare ");
```

Comment:
You shouldn't have to do that here, especially since it's going to impact non-TS nodes.